index.html

```diff
@@ -249,79 +249,74 @@ <h1>Introduction</h1>
 a wide variety of contexts. They serve as communications addresses (telephone
 numbers, email addresses, usernames on social media), ID numbers (for passports,
 drivers licenses, tax IDs, health insurance), and product identifiers (serial
-numbers, barcodes, RFIDs). Resources on the Internet are identified by globally
-unique identifiers in the form of <abbr title="Media Access Control">MAC</abbr>
-addresses; URIs (Uniform Resource Identifiers) are used for resources on the Web
-and each web page you view in a browser has a globally unique URL (Uniform
-Resource Locator).
+numbers, barcodes, RFIDs). URIs (Uniform Resource Identifiers) are used for
+resources on the Web and each web page you view in a browser has a globally
+unique URL (Uniform Resource Locator).
     </p>
     <p>
 The vast majority of these globally unique identifiers are not under our
 control. They are issued by external authorities that decide who or what they
 identify and when they can be revoked. They are useful only in certain contexts
-and recognized only by certain bodies (not of our choosing). They may disappear
-or cease to be valid with the failure of an organization. They may unnecessarily
-reveal personal information. And in many cases they can be fraudulently
-replicated and asserted by a malicious third-party ("identity theft").
+and recognized only by certain bodies not of our choosing. They might
+disappear or cease to be valid with the failure of an organization. They might
+unnecessarily reveal personal information. In many cases, they can be
+fraudulently replicated and asserted by a malicious third-party, which is
+more commonly known as "identity theft".
     </p>
     <p>
 The Decentralized Identifiers (DIDs) defined in this specification are a new
-type of globally unique identifier designed to enable individuals and
-organizations to generate our own identifiers using systems we trust, and to
-prove control of those identifiers (authenticate) using cryptographic proofs
-(for example, digital signatures).
+type of globally unique identifier. They are designed to enable individuals and
+organizations to generate our own identifiers using systems we trust. These
+new identifiers enable us to prove control of over them by authenticating
+using cryptographic proofs such as digital signatures.
```

Comment:
```suggestion
new identifiers enable users to prove control of over them by authenticating
using cryptographic proofs such as digital signatures.
```

Reply:
I agree with not using the first person, but would prefer to avoid "users" when we could say things like "people" or more specific roles.

Reply:
entities?

Reply:
Updated with "entities".